src/app/phase-team-response/issues-pending/issues-pending.component.ts

```diff
@@ -58,10 +58,9 @@ export class IssuesPendingComponent implements OnInit, OnChanges {
   }
 
   ngOnInit() {
-    this.filter = (issue: Issue) => {
-      return (!this.issueService.hasTeamResponse(issue.id) || (!issue.status || issue.status === 'Incomplete')) &&
-        !issue.duplicateOf;
-    };
+    const isNotDuplicate = (issue: Issue) => !issue.duplicateOf;
+    const doesNotHaveResponse = (issue: Issue) => (!this.issueService.hasTeamResponse(issue.id) || (!issue.status || issue.status === 'Incomplete'));
```

Comment:
I realised `doesNotHaveResponse` might not be the best name, since an issue can be considered pending if it has a team response **but** an incomplete status.

How about `doesNotHaveFinalisedResponse` ?

Reply:
Yes, I think that would be better too!